lib/src/components/navigation bar/navigation_bar.dart

```diff
@@ -107,16 +111,25 @@ class ZetaNavigationBar extends ZetaStatelessWidget {
   /// {@macro zeta-widget-semantic-label}
   final String? semanticLabel;
 
+  /// When set to true the items will shrink to take up less space.
+  /// This is useful when there are many items in the navigation bar.
+  /// Defaults to false.
+  /// When set to false the items will take up equal space and will expand to fill the bar.
+  final bool shrinkItems;
```

Comment:
Can you write a test to make sure this works please

Reply:
I was actually going to do the tests in another branch because there were some other issues in other tests and I figured it would be easier for me to do all of that in one branch